packages/element/src/react.js

```diff
@@ -109,6 +111,19 @@ export { isValidElement };
  */
 export { StrictMode };
 
+/**
+ * The React.lazy function lets you render a dynamic import as a regular component.
+ *
+ * @param {function} loadFunction A function that must call a dynamic import(). This must return a Promise which
+ *                                resolves to a module with a default export containing a React component.
+ */
+export { lazy };
```

Comment:
@nosolosw had a good idea to add the link to the original function description in React docs. See #14425.

Reply:
Nice, i updated my PR to use this format in https://github.com/WordPress/gutenberg/pull/14412/commits/bb22466bb98044a340f6d1d8e0b5acd42005a31a